dmsa/service.py

```diff
@@ -111,7 +161,8 @@ def create_erd_route(model, version):
     ext = request.args.get('format') or 'png'
 
     filename = '{0}_{1}_dms_{2}_dmsa_{3}.{4}'.format(
-        model, version, app.config['service_version'], __version__, ext)
+        model, version, get_service_version(app.config['service']), __version__,
```

Comment:
Can the `get_service_version` call be moved into `refresh_data_models_template`, instead?

Reply:
I'll check it out. Another thing about service version is that it is sent with every response, so strictly speaking it's not needed to make a separate request for that.

Reply:
But I don't really feel like doing the refactorings in utility.py and changing the unit tests, so I'll just do as you suggest.